index.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Assignment sollution for Module 2</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Alice&display=swap" rel="stylesheet">
+   </head>
+   
+<body>
+<div>
+  <h1>Our Menu</h1>
+   <div>
+   <section>
+   <p1 id="sectionchicken"> <div id="boxchicken">Chicken</div>Lore ipsum dolor sit amet, consecutur adipiscing sed do ut labore. Ut enim ad minim beniam quis nostrud exercitaiton ulicamo mini veniam laboris nisi ex ea commodo consequat.</p1>
```

Comment:
Divs can't be inside paragraphs, see this explanation: https://stackoverflow.com/a/34683474/1012684
Either replace the div with a span or replace the paragraph with a block level element such as a div.

Reply:
Noted, thank you I have done the correction. What a lot a learning I got!!!